tests/unit/local/init/test_init.py

```diff
@@ -4,26 +4,28 @@
 from cookiecutter.exceptions import CookiecutterException
 from samcli.local.init import generate_project
 from samcli.local.init import GenerateProjectFailedError
-from samcli.local.init import RUNTIME_TEMPLATE_MAPPING
+from samcli.local.init import RUNTIME_DEP_TEMPLATE_MAPPING
 
 
 class TestInit(TestCase):
 
     def setUp(self):
         self.location = None
         self.runtime = "python3.6"
+        self.dependency_manager = "pip"
```

Comment:
What about the `None` case?

Reply:
Will add a test for this, if a dependency manager is not given, we default to our choice.